solr_conf/conf/solrconfig.xml

```diff
@@ -11,10 +11,10 @@
   <directoryFactory name="DirectoryFactory" class="${solr.directoryFactory:solr.StandardDirectoryFactory}"/>
 
   <!-- solr lib dirs -->
```

Comment:
No your correct, it seems like jettywrapper does copy solr/fedora configs out of certain directories: 
https://github.com/samvera/jettywrapper/blob/master/tasks/jettywrapper.rake

```